cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -101,83 +101,83 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<functions-size-of-pattern-expression,Size of pattern expression>>  | Scalar   | Returns the number of sub-graphs matching the pattern expression.
 |<<functions-size-of-string,Size of string>>  | Scalar          | Returns the size of a string.
-|<<functions-split,split()>>                    | String            | Returns the sequence of strings which are delimited by split patterns.
+|<<functions-split,split()>>                    | String            | Returns a list of strings resulting from the splitting of the original string around matches of a given delimiter.
 |<<functions-sqrt, sqrt()>>                     | Logarithmic       | Returns the square root of a number.
-|<<functions-startnode,startNode()>>            | Scalar            | Returns the first node of a relationship.
+|<<functions-startnode,startNode()>>            | Scalar            | Returns the start node of a relationship.
 |<<functions-stdev,stDev()>>                    | Aggregating       | Returns the standard deviation for a given value over a group for a sample of a population.
 |<<functions-stdevp,stDevP()>>                  | Aggregating       | Returns the standard deviation for a given value over a group for an entire population.
-|<<functions-substring,substring()>>            | String            | Returns a substring of the original, with a 0-based index start and length.
-|<<functions-sum,sum()>>                        | Aggregating       | Returns the sum of a numeric column.
-|<<functions-tail,tail()>>                      | List              | Returns all but the first element in a list.
-|<<functions-tan, tan()>>                       | Trigonometric     | Returns the tangent of the expression.
+|<<functions-substring,substring()>>            | String            | Returns a substring of the original string, beginning  with a 0-based index start and length..
+|<<functions-sum,sum()>>                        | Aggregating       | Returns the sum of a set of non-`null` numeric values.
+|<<functions-tail,tail()>>                      | List              | Returns a list `l~result~` containing all the elements, excluding the first one, from a list `list`.
+|<<functions-tan, tan()>>                       | Trigonometric     | Returns the tangent of a number.
 |<<functions-timestamp,timestamp()>>            | Scalar            | Returns the difference, measured in milliseconds, between the current time and midnight, January 1, 1970 UTC.
-|<<functions-toboolean,toBoolean()>>                | Scalar            | Converts the argument to a boolean and returns the result.
-|<<functions-tofloat,toFloat()>>                | Scalar            | Converts the argument to a float and returns the result.
-|<<functions-tointeger,toInteger()>>                    | Scalar            | Converts the argument to an integer and returns the result.
+|<<functions-toboolean,toBoolean()>>                | Scalar            | Casts a string value to a boolean value.
```

Comment:
This change makes it more concise! Great to drop the pointless 'returns the result'

Reply:
Although I think your use of 'cast' here and below is perhaps less ideal than 'converts'. I seem to think of casting as something that can be done without 'coercion' or 'conversion', but in these functions, we are doing something more complex, and so 'conversion' or at least 'coercion' would fit better.